# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/VideoEncoder.cpp
# Commit: 8436bfea59b9
# Full Hash: 8436bfea59b95daf271ba8257e8e0e292c9f7372
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - Include ImageContainer.h in VideoEncoder.cpp r=padenot
#   
#   `mozilla::layers::Image` is incomplete type since it's forward declared
#   in MediaData.h, unless `ImageContainer.h` is included.
#   
# ==============================================================================

diff -r 6ecbca91effe -r 8436bfea59b9 dom/media/webcodecs/VideoEncoder.cpp
--- a/dom/media/webcodecs/VideoEncoder.cpp	Fri Dec 22 21:39:23 2023 +0000
+++ b/dom/media/webcodecs/VideoEncoder.cpp	Fri Dec 22 21:39:23 2023 +0000
@@ -11,6 +11,7 @@
 #include "mozilla/dom/VideoFrame.h"
 
 #include "EncoderTraits.h"
+#include "ImageContainer.h"
 #include "VideoUtils.h"
 #include "mozilla/Assertions.h"
 #include "mozilla/Logging.h"
